packages/sdk/src/server/__tests__/server.test.ts

```diff
@@ -687,4 +656,155 @@ describe("ServerClient", () => {
       );
     });
   });
+
+  describe("invokeWorkflowForExternalUser", () => {
+    let client: ServerClient;
+
+    beforeEach(() => {
+      fetchMock.resetMocks();
```

Comment:
Keeping `fetchMock.resetMocks()` here does its job but personally I prefer to setup the mocks in `beforeEach` and do cleanup in `afterEach`.

Reply:
agreed, fixed